src/common/ipcChannels.js

```diff
@@ -0,0 +1,19 @@
+export default {
```

Comment:
this is _super super_ nit picking, but can we kebab case this instead of camel case?